apps/web/app/(navigation)/SideNavigationBar.tsx

```diff
@@ -136,31 +139,50 @@ export default function SideNav() {
     ),
   };
 
+  //edit navbarItems below
+  const NavbarItems = [
```

Comment:
it is just a variable, I don't think it needs capital letter in front

```suggestion
  const navbarItems = [
```